babycenter/templates/core/tags/footerpage.html

```diff
@@ -1,9 +1,9 @@
 {% load wagtailcore_tags %}
 
 {% if footers %}
-  <ul class="footer-links">
+  <ul class="footer-links l-indent">
     {% for page in footers %}
-    <li><a href="{% pageurl page %}">{{page.title}}</a></li>
+    <li class="footer-link--item"><a href="{% pageurl page %}">{{page.title}}</a></li>
```

Comment:
The BEM syntax is incorrect because it's for a modifier `--`, you're making an element `__` in block `footer-links`

So this class should be named `footer-links__item,` or you can just skip that and create a new block named `footer-link.`

In your CSS you do not have a selector for `footer-link--item` either.

I feel like these footer links may be more complicated than required, let's chat tomorrow on a call.